src/res.rc

```diff
@@ -7,9 +7,29 @@
 
 ********************************************************************/
 
+#define APSTUDIO_HIDDEN_SYMBOLS
 #include "winfile.h"
+#include <prsht.h>
+#undef APSTUDIO_HIDDEN_SYMBOLS
 #include "wfcopy.h"
 
+
+#ifdef WIN32
+#undef WIN32
+#endif
+#define WIN32 100
+
+#ifdef WINVER
+#undef WINVER
+#endif
+#define WINVER 0x0400
+
+#ifdef _WIN32_WINNT
+#undef _WIN32_WINNT
+#endif
+#define _WIN32_WINNT 0x0403

```

Comment:
Windows XP is not a supported platform by Microsoft I do not believe this is a valid change

Reply:
This was added to suppress the _WIN32_WINNT redefinition warning. The value was in the resource related section (resource predefinitions) in the project file.
They are most likely not needed because the recent change (commit: d0493ea) got rid of them and made it to use higher scoped corresponding values.

I will revert this commit along with other commits which were addressed by the latest changes.